src/rules/space-before-function-parentheses.md

```diff
@@ -7,11 +7,11 @@ related_rules:
 - space-return-throw-case
 ---
 
-Enforces consistent spacing before opening parenthesis in function definitions.
+在函数定义中打开小括号之前执行一致的间距。
 
-(removed) This rule was **removed** in ESLint v1.0 and **replaced** by the [space-before-function-paren](space-before-function-paren) rule. The name of the rule changed from "parentheses" to "paren" for consistency with the names of other rules.
+（已移除）此规则在 ESLint v1.0 中移除并被 [space-before-function-paren](space-before-function-paren) 规则。该规则的名称从“parentheses”改为“paren”，以便与其他所取代的名称一致。
```

Comment:
```suggestion
（已移除）此规则在 ESLint v1.0 中移除并被 [space-before-function-paren](space-before-function-paren) 规则。该规则的名称从 “parentheses” 改为 “paren”，以便与其他所取代的名称一致。
```

Reply:
Chinese double quotes do not add spaces before and after, we generally add spaces between Chinese and English, numbers, and in-line code.

Reply:
oh okay got it.